### PR TITLE
name change et al

### PR DESCRIPTION
**Pull request by doktordirk**

### CODE REVIEW

**Comment by doktordirk** [comment]
@RWOverdijk this (ps: add label to all repos)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # aurelia-api
 
-[![Build Status](https://travis-ci.org/SpoonX/aurelia-api.svg?branch=master)](https://travis-ci.org/SpoonX/aurelia-api)
+[![Build Status](https://travis-ci.org/aurelia-api.svg?branch=master)](https://travis-ci.org/SpoonX/aurelia-api)
```
Comment (RWOverdijk) [inline README.md]: I think your search/replace went one step too far :dancers:

File: README.md [hunk 2]
```diff
@@ -9,11 +9,15 @@ This library plays nice with the [Sails.js framework](http://sailsjs.org).
 
 > To keep up to date on [Aurelia](http://www.aurelia.io/), please visit and subscribe to [the official blog](http://blog.durandal.io/). If you have questions, we invite you to [join us on Gitter](https://gitter.im/aurelia/discuss). If you would like to have deeper insight into our development process, please install the [ZenHub](https://zenhub.io) Chrome Extension and visit any of our repository's boards. You can get an overview of all Aurelia work by visiting [the framework board](https://github.com/aurelia/framework#boards).
 
+## Important note
+
+This plugin is now registered with npm and jspm as `aurelia-api`. This simplifies the installation. Other spoonx plugins, particularly [aurelia-authentication](https://github.com/spoonx/aurelia-authentication) and [aurelia-orm](https://github.com/spoonx/aurelia-orm), will also now reference to this plugin by `aurelia-api` only (loosing the `spoonx/..` prefix). You will need to update your project's package.json accordingly to prevent forks.
```
Comment (RWOverdijk) [inline README.md]: Simplify like other repos

File: doc/rest.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Docs for {`Rest`} client
 
 ```javascript
-import {Rest} from 'spoonx/aurelia-api';
+import {Rest} from 'aurelia-api';
```
Comment (RWOverdijk) [inline doc/rest.md]: These changes already reside here: http://aurelia-api.spoonx.org/api_rest.html I think we can just remove the docs

**Comment by RWOverdijk** [comment]
Labels added, and Minor feedback. After that :+1:

**Pull request merged**